.github/workflows/build-manywheel-images.yml

```diff
@@ -176,7 +176,7 @@ jobs:
           .ci/docker/manywheel/build.sh manylinux-builder:rocm${{matrix.rocm_version}}
   build-docker-cpu:
     environment: ${{ (github.ref == 'refs/heads/main' || startsWith(github.event.ref, 'refs/tags/v')) && 'docker-build' || '' }}
-    runs-on: linux.9xlarge.ephemeral
+    runs-on: am2.linux.9xlarge.ephemeral
```

Comment:
this job is running just fine. please don't switch it back

Reply:
https://github.com/pytorch/pytorch/actions/runs/10476660034/job/29029704836 does not look like that is the case.

Reply:
CPU jobs is failing here: https://github.com/pytorch/pytorch/actions/runs/10476660034/job/29016049984

Reply:
Have we checked the AMI's yum/repo? The URLs seem to make a huge difference in terms of downloading speed. e.g. are those runners from a low-network-bandwidth AWS region, or trying to access the URLs that are impossible to provide good downloading speed?